setup.py

```diff
@@ -23,10 +24,13 @@
             'postgis = intake_geopandas.geopandas:PostGISSource',
             'shapefile = intake_geopandas.geopandas:ShapefileSource',
             'spatialite = intake_geopandas.geopandas:SpatiaLiteSource',
+            'regionmask = intake_geopandas.regionmask:RegionmaskSource',
         ]
     },
     include_package_data=True,
     install_requires=requires,
+    extra_require=['regionmask'],
```

Comment:
This is still a typo, should be `extras_require`

Reply:
ping on this comment @aaronspring

Reply:
sorry. didnt see